### PR TITLE
OP-483 pass 1 of documentation changes

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-483.   Pass 1 of many passes updating the doc for GUI changes.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -65,31 +65,31 @@ In this document the following conventions are used:
...
 
 * When you find a text written in bold as - *Search patient visits –* it indicates a function of the application,
 or an area of the window (for example, *Data table*).
 
 * Each button always has a single letter with an underscore. You can select the functionality offered by the button by
-pressing the “Alt” key and at the same time the “underscored” key (in the example of *[.underline]##P##harmacy* you
-press “Alt” and “P”)*.* This behavior is common all over the application and allows the user to operate (almost)
-without the use of the mouse. In the entire document the “Alt” key and the “P” key will be indicated as “Alt + P”.
+pressing the “Alt” key and at the same time the “underlined” key (in the example of *[.underline]##P##harmacy* you
+press “Alt” and “P”)*.* This behavior is common thoughout the application and allows the user to operate (almost)
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: missing an "r" in  "throughout"?

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -65,31 +65,31 @@ In this document the following conventions are used:
...
-area. Areas can be a *Selection panel*, a *Data table*, or a *Buttons panel* (see *_Laboratory browsing_* window below).
+area. Areas can be a *Selection panel*, a *Data table*, or a *Buttons panel* (see *_Laboratory Browser_* window below).
 
-image:image3.png[../../Screen%20Shot%202019-07-26%20at%2015.15.43.png,width=642,height=464]
+image:image3.png[Window areas,width=642,height=464]
 
-* To highlight a record (for example a single line in a *Data table*) you click the mouse on the record.
-The record (the entire line) is then highlighted in blue. This is valid all over the application (see example below):
+* To highlight an entry (for example, a single line in a *Data table*) you click the mouse on the entry.
+The entry (the entire line) is then highlighted in blue. This behavior is valid thoughout the application (see example below):
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: missing an "r" in "throughout"?

File: doc_user/UserManual.adoc [hunk 3]
```diff
@@ -154,19 +154,19 @@ The following fields are available to select or narrow the resulting matches:
 * **Date. “**DATE From” and “DATE To” allows for the selection of all visits performed during the specified period
 * **Age. “**Age From” and “Age To” allows for the selection of the patients by age
 * *Sex.* Allows for the selection of all the patients by sex: All / Male / Female
-* *Type of patient.* Allows for the selection of all the visits by type of attendance: All / New / Re-Attendance
+* *Type of patient.* Allows for the selection of all the visits by type of attendance: All / New / Readmittance
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: Here the term "Re-Attendance" is the one commonly used to refer to OPD cases: it indicates a subsequent (visit) concerning the same medical case (same disease) or a planned followup.

File: doc_user/UserManual.adoc [hunk 4]
```diff
@@ -193,9 +193,9 @@ The second and third list still contains all diseases.
  +
  +
 _Example: Disease Types can be defined by the Administrator. Ask your Administrator how to do it or see
-<<general-data,General Data>> in this document._
+<<settings,Settings>> in this document._
  +
-* *Diagnosis.* A maximum of three diseases can be diagnosed per each attendance (“new attendance” or “re-attendance”
+* *Diagnosis.* A maximum of three diseases can be diagnosed per each attendance (“new attendance” or “readmittance”
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: Here the term "Re-Attendance" is the one commonly used to refer to OPD cases: it indicates a subsequent (visit) concerning the same medical case (same disease) or a planned followup.

File: doc_user/UserManual.adoc [hunk 5]
```diff
@@ -256,7 +256,7 @@ The OPD Extended module affords the following improvements:
 * Calendar to choose the visit date (1);
 * OPD number now is visible (2): it is the progressive number per year automatically set by the system, one can change the value needed. The OPD number MUST be unique for each year;
 * Registered Patient (3): before choosing a disease, you MUST first select a patient from the drop-down list; you can even fast search the patient with the search field nearby by typing part of its name or code; this will improve reliability for reports and statistics, and thus service to the patients by knowing his/her history.
-* Once the patient is selected and a previous OPD visit is found, special fields will be populated with the most recent visit of the patient (4), thus his/her personal data (5), the re-attendance checkbox will be selected automatically (6);
+* Once the patient is selected and a previous OPD visit is found, special fields will be populated with the most recent visit of the patient (4), thus his/her personal data (5), the readmittance checkbox will be selected automatically (6);
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: Here the term "Re-Attendance" is the one commonly used to refer to OPD cases: it indicates a subsequent (visit) concerning the same medical case (same disease) or a planned followup.

File: doc_user/UserManual.adoc [hunk 6]
```diff
@@ -803,19 +801,19 @@ press the *[.underline]##D##elete* button to see the confirmation window shown b
 
 image:image48.png[image,width=240,height=174]
 
-=== 7.3 Laboratory Extended (v.1.1)
+=== 7.3 Laboratory Extended
 
-Since Open Hospital version 1.3.1 you can extend the _New Laboratory_ functionality by changing the _LABEXTENDED_ flag in the configuration file. Ask your Administrator how to do it.
+The Lab functionality cam be extended by changing the _LABEXTENDED_ flag in the configuration file. Ask your Administrator how to do it.
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: "can" in place of "cam"?

File: doc_user/UserManual.adoc [hunk 7]
```diff
@@ -1325,11 +1323,11 @@ Doing the selection, data about patients can be selected by choosing specific fi
...
 
 NOTE: Wards can be defined by the Administrator. Ask your Administrator how to do it or see
-<<general-data,General Data>> in this document.
+<<settings,Settings>> in this document.
 
-====  10.2.2 Search patient Enhanced
+====  10.2.2 Search Patient Enhanced
 
-Open Hospital, since version 1.6, allows optimizing memory usage when the number of registered patients becomes huge, by changing the _ENHANCEDSEARCH_ flag in the configuration file. Ask your Administrator how to do it or read the _Administrator’s Guide_.
+One cah optimize memory usage when the number of registered patients becomes huge, by changing the _ENHANCEDSEARCH_ flag in the configuration file. Ask your Administrator how to do it or read the _Administrator’s Guide_.
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: "can" in place of "cah"

**Review by mwithi — CHANGES_REQUESTED** [review]
Just few orthography. Thanks!

File: doc_user/UserManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Fixed; thanks

File: doc_user/UserManual.adoc (hunk 2 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Fixed; thanks

File: doc_user/UserManual.adoc (hunk 4 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Readmittance is a more appropriate English word for "re-attendance" (which BTW is only correct without the "hyphen") and I have been changing it in the resource bundle files.

File: doc_user/UserManual.adoc (hunk 5 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Readmittance is a more appropriate English word for "re-attendance" (which BTW is only correct without the "hyphen") and I have been changing it in the resource bundle files.

File: doc_user/UserManual.adoc (hunk 6 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Got it; thanks

File: doc_user/UserManual.adoc (hunk 7 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Ugh; fixed this one too.

File: doc_user/UserManual.adoc (hunk 3 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Readmittance is a more appropriate English word for "re-attendance" (which BTW is only correct without the "hyphen") and I have been changing it in the resource bundle files.

Comment (mwithi) [reply doc_user/UserManual.adoc]: please consider this term as not-questionable at the moment: we'll use Re-Attendance (with hyphen) when we deal with OPDs.
Readmittance could be used for subsequent Admissions, but they are not yet labelled as such in the application (yet).

Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Done

File: doc_user/UserManual.adoc (hunk 5 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: done

File: doc_user/UserManual.adoc (hunk 4 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: done

**Review by mwithi — APPROVED** [review]

**Pull request merged**